Doc Queue item 4: Add text about PCRE regex support to filter disks. (#689)
freenas/freenas-docs: userguide/storage.rst

Choosing a name that will stick out in the logs is recommended,
rather than generic names like "data" or "freenas".

To encrypt data on the underlying disks as a protection against physical
theft, set the :guilabel:`Encryption` option. A pop-up message shows a

> reminder to :literal:`Always back up the key!`. Without the key, the

Reviewer: ```suggestion
reminder to :literal:`Always back up the key!`. The data on
```

Reviewer: flow

> data on the disks is inaccessible. Select :guilabel:`Confirm` then click

Reviewer: ```suggestion
the disks is inaccessible without the key. Select :guilabel:`Confirm` then click
```
Flow.

:guilabel:`I UNDERSTAND`.

.. warning:: Refer to the warnings in :ref:`Managing Encrypted Pools`
   before enabling encryption!

> In the :guilabel:`Available Disks` section, select disks to add to the

Reviewer: ```suggestion
Select disks to add to the pool In the
```

Reviewer: flow

Reviewer 2: s/In the/from the/

> pool. Enter a value in :guilabel:`Filter disks by name` or

Reviewer: ```suggestion
 :guilabel:`Available Disks` section. Enter a value in :guilabel:`Filter disks by name` or
```

Reviewer: flow

:guilabel:`Filter disks by capacity` to change the displayed disk order.
These fields support
`PCRE regular expressions <http://php.net/manual/en/reference.pcre.pattern.syntax.php>`__
for filtering. For example, to show only *da* and *nvd* disks in
:guilabel:`Available Disks`, type :literal:`^(da)|(nvd)` in
:guilabel:`Filter disks by name`.

> After selecting the desired disks, click the right arrow to add them

Reviewer 2: ```suggestion
After selecting disks, click the right arrow to add them
```

to the :guilabel:`Data VDevs` section. The usable space of each disk in

> a pool is limited to the size of the smallest disk in the pool. Because

Reviewer 2: ```suggestion
a pool is limited to the size of the smallest disk in the vdev. Because
```

of this, creating pools with the same size disks is recommended.

Any disks that appear in :guilabel:`Data VDevs` are used to create the

> pool. To remove a disk from that section, select the disk and use the

Reviewer 2: ```suggestion
pool. To remove a disk from that section, select the disk and click the
```
